go.mod

```diff
@@ -1,3 +1,3 @@
-module never18
+module github.com/kachick/never18
```

Comment:
Error: cmd/never18/main.go:7:2: package never18 is not in GOROOT (/opt/hostedtoolcache/go/1.20.5/x64/src/never18) (compile)

staticcheck error came from any conflict of this module changing? 🤔